src/databricks/labs/ucx/hive_metastore/catalog_schema.py

```diff
@@ -71,6 +104,16 @@ def _list_target(self) -> tuple[set[str], dict[str, set[str]]]:
             target_schemas[target_catalog].add(target_schema)
         return target_catalogs, target_schemas
 
+    def _get_src_database_target_mapping(self) -> dict[str, dict]:
+        """generate a list of catalogs and schema to be created from table mappings."""
+        src_trg_schema_mapping: dict[str, dict] = {}
```

Comment:
What is trg? dst?

Reply:
its the mapping from src hivemetastore schema to its target uc catalog and UC schema. have added a comment to make it clear